### PR TITLE
DOCS-1462: added-odoo

### PR DESCRIPTION
**Pull request by Alex-MultiSafepay**
### Basic
- We only accept documentation that is proved to work on our LIVE API
- At least 1 approval is needed before pull requests can be merged
- The article must match our Synonym word list. For external commits, a MultiSafepay employee will check
- US English must be applied
- Use bullets for numerations where the sort order doesn't matter. For chronological order, use numbers. This does not apply to changelogs
- Set link on email address.

### Text style
- Paths are written like: _Configuration → System → Plugin → MultiSafepay_
- Use single quotation marks in your Markdown code (example title: 'Title of the article')
- Use Italics for buttons, clicks, paths, etc. Do mind to close the italics before the end of a sentence followed by a period
- Numerations / bullets are punctuated with a period at the end of the numerations / bullets.
- Periods should be excluded for sentences that finish in an e-mail address (e.g. "..contact us at <example@example.com> "  )

### Images
- Screenshot recomended sizes: Width: 820px - Height: auto

### CODE REVIEW

File: content/integrations/plugins/odoo/faq/available-payment-methods.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title : "Available payment methods in Odoo"
+meta_title: "Magento 2 plugin available payment methods - MultiSafepay Documentation Center"
```
Comment (Qadira-MultiSafepay) [inline content/integrations/plugins/odoo/faq/available-payment-methods.md:3]: Magento 2 = Odoo

File: content/integrations/plugins/odoo/manual.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+title : "MultiSafepay Magento 2 installation & configuration manual"
+meta_title: "Magento 2 plugin manual - MultiSafepay Documentation Center"
```
Comment (Qadira-MultiSafepay) [inline content/integrations/plugins/odoo/manual.md:3]: Magento 2 = Odoo

File: content/integrations/plugins/odoo/manual.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for Payment Methods, Tools and General Questions as well as the contact details of our Support and Integration Teams."
+aliases:
+    - /support-tab/magento2/manual
```
Comment (Qadira-MultiSafepay) [inline content/integrations/plugins/odoo/manual.md:6]: Please do a general search and change M2 in Odoo

File: content/integrations/plugins/odoo/manual.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+### 1. Requirements
+- Odoo 13.0
+- Tested on Python 3.6
+
+### 2. Installation
+1. Download ZIP archive with module
+2. Unpack the content of the .ZIP file and add _payment_multisafepay_official_ folder under your custom apps in your Odoo server
+   (`/mnt/extra-addons/`)
+3. Install the required Python dependencies.
```
Comment (Qadira-MultiSafepay) [inline content/integrations/plugins/odoo/manual.md:17]: remove dot

File: content/integrations/plugins/odoo/manual.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    Alternatively, you can install them manually by doing
+    ```shell
+    pip3 install multisafepay==0.2.0
+    ```
+    You can find all the information about the dependencies at this [link](https://pypi.org/project/multisafepay/)
+4.  Restart Odoo server
+5.  Activate developer mode at Odoo platform
+6.  Navigate to Apps menu and click _Update Apps List_
+7.  Search and open MultiSafepay payments module (payment_multisafepay_official)
+8.  Click _Install_
```
Comment (Qadira-MultiSafepay) [inline content/integrations/plugins/odoo/manual.md:30]: Add dot

File: content/integrations/plugins/odoo/manual.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+For these payment methods, if the order was created not in euros, the amount will be _converted_ to the required currency. 
+_Odoo platform currency rate_ is used for conversions and it is configured by a system administrator.
+
+### 4. Congratulations
+You have installed and configured the plugin successfully. If you have any questions regarding the plugin, feel free to contact our Integration Team at <integration@multisafepay.com>
+
+### 5. Updates 
+1.  Navigate to _Apps_ menu
+2.  Search and open _MultiSafepay payments_ module
+3.  Click _Upgrade_
```
Comment (Qadira-MultiSafepay) [inline content/integrations/plugins/odoo/manual.md:58]: Add dot

File: content/integrations/plugins/odoo/manual.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+### 3. Configuration
+Configuration is possible within the _Invoicing_ menu or the _Website_ menu.
+1.  Navigate to _Invoicing/Website_ menu and go to _Configuration_ -> _Payment Acquirers_
+2.  Select _MultiSafepay_ payment acquirer. Go to _Edit_ mode. By default acquirer state is disabled.
+3.  Change state, _Enabled_ or _Test_, and enter _Live_ and/or _Test_ _API Key_, set Journal at Configuration tab, and save changes.
+4.  Click _Pull payment methods_ at Configuration tab to get payment methods from your MultiSafepay account.
+    
+    Be sure that you have the desired payment methods active in your [MultiSafepay Control](https://testmerchant.multisafepay.com)
+5.  Configure payment methods you wish to offer. Each payment method will have to be configured separately.
```
Comment (Qadira-MultiSafepay) [inline content/integrations/plugins/odoo/manual.md:41]: Closing is only at the end of the list with a period.
Please remove all dots in enumeration and close at #5 with a period

**Review by Qadira-MultiSafepay — CHANGES_REQUESTED** [review]

**Comment by Alex-MultiSafepay** [comment]
@Qadira-MultiSafepay all changes done

File: content/integrations/plugins/odoo/manual.md (hunk 7 above)
Comment (Alex-MultiSafepay) [reply content/integrations/plugins/odoo/manual.md:41]: now everything should be ok

**Pull request merged**